apps/src/templates/Button.jsx

```diff
@@ -303,6 +313,18 @@ const styles = {
       paddingRight: 10,
       lineHeight: '40px'
     }
+  },
+  textButton: {
+    color: color.teal,
+    border: 'none',
+    backgroundColor: 'unset',
+    fontWeight: 'bold',
```

Comment:
unfortunately, i think we may actually need to change the `fontFamily` as well as the `fontWeight` here. i've usually seen our bold styles use `fontFamily: '"Gotham 5r", sans-serif'`. normally, this will mean we just use "Gotham 5r," but if we fallback to sans-serif, we'll still get the bold weighting.